src/main/java/com/cburch/logisim/std/arith/ShifterHDLGeneratorFactory.java

```diff
@@ -124,10 +124,10 @@ public void getGenerationTimeWiresPorts(Netlist theNetlist, AttributeSet attrs)
             """, nrOfShiftBits - 1);
       }
     }
-    return contents.getWithIndent();
+    return contents;
   }
 
-  private ArrayList<String> GetStageFunctionalityVerilog(int stageNumber, int nrOfBits) {
+  private ArrayList<String> getStageFunctionalityVerilog(int stageNumber, int nrOfBits) {
```

Comment:
Just wonder if it'd not be more consistent to have this (`getStageFunctionalityVhdl()` too) return LineBuffer too?

Reply:
I'll look into it in a new PR, I do not want to block yours more

Reply:
It was just a note/comment, hence I approved regardless of your answer.